pyrdp/ui/qt.py

```diff
@@ -24,10 +24,10 @@
 QRemoteDesktop is a widget use for render in rdpy
 """
 
+import rle
 from io import BytesIO
 
-import rle
```

Comment:
FYI: `rle` is not an stdlib module. Your import organizer is broken and you shouldn't have made that change.